### ISSUE TITLE
NO_KEY mode crashes the game

### ISSUE THREAD

**Issue by TechPro424** [Daxanius/NoPryingEyes#32]
https://discord.gg/UuKQasSUrn/
This link from Modrinth seems to have expired, I'm unable to join the Discord

Also, there's a crash when I try to join the PvPLegacy MC server on 1.20.2 with signing mode set to `NO_KEY`
https://mclo.gs/jpeix5p
Does not occur with signing mode `NO_SIGN`

**Comment by TechPro424** [comment]
tried to join MCC Island server with NO_KEY
Here's the crash: https://mclo.gs/hJnB4Cx/

**Comment by Daxanius** [comment]
Yeah, I'm aware of the NO_KEY crash, I've resolved it in the dev branch, there's 2 more issues I want to resolve before updating NPE for all Minecraft versions it supports. At least the ones where it broke, it seems updating NPE broke this system somewhere down the line. 

I deleted the Discord from before because I was looking into deleting my Discord account, but after a while I decided it wasn't worth it. In any case, I have recreated a Discord server and updated the Modrinth page with the new invite. I encourage you to join the server.
